README.md

```diff
@@ -14,6 +14,7 @@ We support:
 
 * [OS X Mavericks (10.9)](https://itunes.apple.com/us/app/os-x-mavericks/id675248567)
 * [OS X Yosemite (10.10)](https://www.apple.com/osx/)
```

Comment:
Same here - I don't think this is the place to find 10.10 now that 10.11 is out. What about just killing these links?